test/messageCheckers.test.js

```diff
@@ -235,102 +236,58 @@ describe('isPrivilegedUser()', function () {
-    };
-
-    const message = '2';
-
-    expect(isValidVote(message, pollState)).to.be.false;
-  });
-
-  it('returns false if the poll is invisible', function () {
-    const pollState = {
-      active: true,
-      visible: false,
-      title: 'Some Title',
-      options: { 1: ' ', 2: ' ' },
-      userVotes: { user1: '1', user2: '2' },
-    };
-
-    const message = '2';
-
-    expect(isValidVote(message, pollState)).to.be.false;
-  });
-
   it('returns false if the message does not start with a separate single digit number', function () {
-    const pollState = {
-      active: true,
-      visible: true,
-      title: 'Some Title',
-      options: { 1: ' ', 2: ' ' },
-      userVotes: { user1: '1', user2: '2' },
-    };
-
     let message = 'some text';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
 
     message = 'some text 2';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
 
     message = '2text';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
 
     message = ' 2text';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
   });
 
   it('returns true if the message starts with a separate single digit number', function () {
-    const pollState = {
-      active: true,
-      visible: true,
-      title: 'Some Title',
-      options: { 1: ' ', 2: ' ' },
-      userVotes: { user1: '1', user2: '2' },
-    };
-
     let message = '9';
-    expect(isValidVote(message, pollState)).to.be.true;
+    expect(isValidVote(message)).to.be.true;
 
     message = '0';
-    expect(isValidVote(message, pollState)).to.be.true;
+    expect(isValidVote(message)).to.be.true;
 
     message = '2';
-    expect(isValidVote(message, pollState)).to.be.true;
+    expect(isValidVote(message)).to.be.true;
 
     message = '99';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
 
     message = '2 test';
-    expect(isValidVote(message, pollState)).to.be.true;
+    expect(isValidVote(message)).to.be.true;
 
     message = '2 test ';
-    expect(isValidVote(message, pollState)).to.be.true;
+    expect(isValidVote(message)).to.be.true;
 
     message = ' 2 test';
-    expect(isValidVote(message, pollState)).to.be.true;
+    expect(isValidVote(message)).to.be.true;
 
     message = ' 2 test ';
-    expect(isValidVote(message, pollState)).to.be.true;
+    expect(isValidVote(message)).to.be.true;
 
     message = '2test';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
 
     message = ' 2test';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
 
     message = ' 2test ';
-    expect(isValidVote(message, pollState)).to.be.false;
+    expect(isValidVote(message)).to.be.false;
```

Comment:
Ich glaube das habe ich vemurkst vorher, aber wollen wir die Lines, die auf `to.be.false` enden unter den anderen Test-Case unter Zeile 239 schieben?

Reply:
klaro!